src/PBot/Issues/CheckIssuesInMyRepos.cs

```diff
@@ -9,7 +9,7 @@
     public class CheckIssuesInMyRepos : BotCommand
     {
         public CheckIssuesInMyRepos()
-            : base("check my repos$", "pbot check my repos - List the issue that needs attention for repos that you are the caretaker for")
+            : base("check my repos$", "`pbot check my repos` - List the issue that needs attention for repos that you are the caretaker for")
```

Comment:
I agree. It's painful to repeat this formatting in every command. We should split the argument in two, e.g. `syntax` and `description`.